### DISCUSSION TITLE
How to change vue router scrollBehaviour

### DISCUSSION THREAD

**Discussion by soerenmartius** [antfu/vitesse#377] [General]
Hi, 

How can I change the [scrollBehaviour](https://router.vuejs.org/guide/advanced/scroll-behavior.html) of the `vue-router`? `ViteSSG` only accepts `RouterOptions` which only accepts `base`.

Thanks in advance!

**Comment by hannoeru** [comment]
Example:

https://github.com/antfu/antfu.me/blob/0bbb2575175b00a290e8b8f12dcb6dc2b773ab43/src/main.ts#L23-L41
